test/plugin/scenarios/spring-6.x-scenario/support-version.list

```diff
@@ -15,3 +15,5 @@
 # limitations under the License.
 
 6.0.4
+6.1.0
+6.1.1
```

Comment:
So, only this is using SpringBoot 3.2.0, which was failing before the fix, am I getting your idea correctly?

Reply:
yes, I tracd my own springboot project in 3.2.0(Spring 6.1.1), and find SpringMVC node missing, but springboot project in 3.1.6(Spring 6.0.14) works.